pkg/cmd/cli/cmd/loginoptions.go

```diff
@@ -372,3 +380,37 @@ func (o *LoginOptions) serverProvided() bool {
 	_, err := o.ClientConfig.ClientConfig()
 	return err == nil || !clientcmd.IsNoServerFound(err)
 }
+
+func relativizeClientConfigPaths(cfg *clientcmdapi.Config, base string) (err error) {
```

Comment:
My first try was to put this as a public helper under `pkg/cmd/config`, however it result in cyclic dependencies when trying to import `pkg/cmd/util`. I only needed `filepath.go`. In fact `pkg/cmd/util` is looking a little loaded, `clientconfig.go` should probably has its own package. I'll address that and Clayton's comment and add a "TODO make upstream" tomorrow.